CONTRIBUTING.md

```diff
@@ -185,27 +185,32 @@ For all available scripts see the package's `package.json` file.
 **Prerequisites:** [Docker][docker] 🐳
 
 You can run visual regression testing via `Makefile` in the project root.
-Using `make test-e2e` you will execute docker command that starts [Playwright][playwright] in containerized environment.
+All commands will execute docker command that starts [Playwright][playwright] in containerized environment.
+
+- Using `make test-e2e` you will run the tests.
+- Using `make test-e2e-update` you will update the snapshots.
+- Using `make test-e2e-report` you will generate and serve report of the visual regression testing.
 
 👉 Visual snapshots are generated based on platform, so we need to use same platform locally and on CI (GitHub Actions).
 
 ⚠️ Version number of the [Playwright][playwright] dependency must be the same in `package.json` file and in the `./bin/make/e2e.sh` to ensure that no additional [Playwright][playwright] dependencies will need to install (browsers are backed in the Docker image). See https://playwright.dev/docs/docker.
 
-⚠️ Visual regression testing currently targets only deploys on Netlify.
-We are investigating how we should handle visual testing and several paths like
+We run visual regression testing locally against our demo apps. Web and Web React packages are served using Vite.
```

Comment:
(I'm commenting on the unchanged paragraph): I think these occurrences of "Playwright" don't need to be links.  It's OK to have the link just in the first occurrence on the page.